src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt

```diff
@@ -95,6 +98,16 @@ object SectionSyncListener : Listener {
         }
     }
 
+    @EventHandler
+    fun BlockPlaceEvent.disableBlocks() {
+        val section = WorldManager.getSectionFor(player.location) ?: return
```

Comment:
Plz add a toggle for this in the config that is off by default.

Reply:
this function can be removed once my changes in MadeInAbyss go through tmorrow